notifications/store.go

```diff
@@ -0,0 +1,42 @@
+package notifications
+
+import (
+	"sync"
+
+	"github.com/textileio/powergate/v2/ffs"
+)
+
+type configStore struct {
+	sync.RWMutex
+
+	configs map[ffs.JobID][]*ffs.NotificationConfig
+}
```

Comment:
Will this need to write to a persistent store (database) at some point?

Reply:
yes, I think at some point would be better to put it to persistent store, for the beginning I guess we can keep it in memory